packages/amplify_datastore/android/src/main/kotlin/com/amazonaws/amplify/amplify_datastore/AmplifyDataStorePlugin.kt

```diff
@@ -672,4 +655,78 @@ class AmplifyDataStorePlugin : FlutterPlugin, MethodCallHandler {
             }
         }
     }
+
+    private fun createErrorHandler(request: Map<String, Any>) : DataStoreErrorHandler{
+        var errorHandler : DataStoreErrorHandler
+        errorHandler = if( (request["hasErrorHandler"] as? Boolean? == true) ) {
+            DataStoreErrorHandler {
+                val args = hashMapOf(
+                        "errorCode" to "DataStoreException",
+                        "errorMessage" to ExceptionMessages.defaultFallbackExceptionMessage,
+                        "details" to createSerializedError(it)
+                )
+                channel.invokeMethod("errorHandler", args)
+            }
+        } else {
+            DataStoreErrorHandler {
+                LOG.error(it.toString())
+            }
+        }
+        return errorHandler
+    }
+
+    private fun createConflictHandler(request: Map<String, Any>) : DataStoreConflictHandler{
+        var conflictHandler: DataStoreConflictHandler
+        conflictHandler = if ((request["hasConflictHandler"] as? Boolean? == true)) {
+            DataStoreConflictHandler { conflictData,
+                                       onDecision ->
+
+                val modelName = conflictData.local.modelName
+                val args = hashMapOf(
+                        "modelName" to modelName,
+                        "local" to FlutterSerializedModel(conflictData.local as SerializedModel).toMap(),
+                        "remote" to FlutterSerializedModel(conflictData.remote as SerializedModel).toMap()
+                )
+
+                uiThreadHandler.post {
+                    channel.invokeMethod("conflictHandler", args, object : Result {
+                        override fun success(result: Any?) {
+                            val resultMap: Map<String, Any>? = result.safeCastToMap()
+                            when (resultMap?.get("resolutionStrategy") as String) {
+                                "APPLY_REMOTE" -> onDecision.accept(DataStoreConflictHandler.ConflictResolutionDecision.applyRemote())
```

Comment:
Could use `DataStoreConflictHandler.ResolutionStrategy` to replace hardcoded string `"APPLY_REMOTE"` for consistency.

Reply:
Changed to use enum